code/datums/gamemode/role/wizard.dm

```diff
@@ -48,7 +48,8 @@
 			switch(faction.name)
 				if("The Manajerks","The Quickvillains")
```

Comment:
Is this text even still being sent, as the names of the factions were changed?

Reply:
I noticed that too and it confused me, i swore I've seen the cwc greeting before. I'll test it real quickly

Reply:
I swapped the old names for the current ones, works fine now.